tests/test_lookml.py

```diff
@@ -560,6 +580,16 @@ def test_lookml_actual(mock_glean_ping, runner, glean_apps, tmp_path, msg_glean_
     with runner.isolated_filesystem():
         with patch("google.cloud.bigquery.Client", MockClient):
             _lookml(open(namespaces), glean_apps, "looker-hub/")
+        yield namespaces_text
+
+
+@patch("generator.views.glean_ping_view.GleanPing")
+def test_lookml_actual_baseline_view(
+    mock_glean_ping, runner, glean_apps, tmp_path, msg_glean_probes
+):
+    with _prepare_lookml_actual_test(
+        mock_glean_ping, runner, glean_apps, tmp_path, msg_glean_probes
+    ) as namespaces_text:
```

Comment:
Refactored the tests so there is a single test per unit instead of one large test. This could be refactored further to reduce boilerplate, but leaving it at this indentation has a benefit of making the diff relatively small.